roles/passenger/tasks/main.yml

```diff
@@ -17,11 +17,10 @@
 
 - name: install passenger-install-apache2-module
   sudo: yes
-  shell: passenger-install-apache2-module --auto creates=/usr/local/lib/ruby/gems/{{ ruby_ver }}/gems/passenger-{{ passenger_ver }}/buildout/apache2/mod_passenger.so 
+  shell: passenger-install-apache2-module --auto creates=/usr/local/lib/ruby/gems/{{ ruby_abi }}.0/gems/passenger-{{ passenger_ver }}/buildout/apache2/mod_passenger.so
```

Comment:
This task had been running every time since the `gems/` versioned subdirectory only changes on minor releases, not patch releases.